MMR.Randomizer/Resources/REQ_CASUAL.txt

```diff
@@ -37,70 +37,51 @@
       ],
       "ConditionalItems": [
         [
-          "ItemHookshot",
-          "Scarecrow's Song"
+          "Use Fire Arrow"
```

Comment:
Any idea why this used to require Goron or Zora Jumps or FD Jumps?

Reply:
Jumping down from the 3rd floor to the Wizrobe used to not be in logic, but I changed that because it's a scenario that only happens in dungeon entrance randomizer anyway, so I decided that the people who play with dungeon entrance randomizer on would want this in logic. There is an entry called SHT 3F to 2F jump, but that is explicitly for jumping to the ramp after the pound switch puzzle, which is a more difficult jump anyway.